apps/dashboard/biome.json

```
@@ -0,0 +1,14 @@
+{
+    "root": false,
```

Biome (1.9.4)

[error] Found an unknown key `root`.

Known keys:


(deserialize)